test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
```

Comment:
> I just said, errors.New communicates better the intend imo.

Agree, but if I need to use `%s`, then it would look like this:
```
errors.New(fmt.Sprintf("This happened: %s, just terrible!", err))
```